packages/itwin/tree-widget/src/components/trees/category-tree/CategoriesTree.tsx

```diff
@@ -54,10 +54,10 @@ export interface CategoryTreeProps extends BaseFilterableTreeProps {
    */
   viewManager?: ViewManager;
   /**
-   * Flag that determines if hierarchy level filtering will be enabled for this tree.
+   * Props for configuring hierarchy level filtering.
    * @beta
    */
-  isHierarchyLevelFilteringEnabled?: boolean;
+  hierarchyLevelFilteringProps?: HierarchyLevelFilteringProps;
```

Comment:
Need a better name for this prop. While it's recommended to have filtering enabled when limiting, the two aren't really related to each other. So putting limiting properties under "filtering" prop is not correct, IMO.

Reply:
Do you suggest not putting them under the same interface and have them as two separate props?